### PR TITLE
Use kube-prometheus-stack chart

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: clusters/kepler.yaml [hunk 1]
```diff
@@ -32,3 +32,5 @@ spec:
       sourceRef:
         kind: HelmRepository
         name: kepler
+  dependsOn:
+    - name: kube-prometheus-stack
```
Comment (rossf7) [inline clusters/kepler.yaml:36]: Adds the dependency on the service monitor CRD

File: clusters/flux-system/gotk-sync.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: prometheus-stack
```
Comment (rossf7) [inline clusters/flux-system/gotk-sync.yaml:11]: I couldn't find a way to keep prometheus deployed without switching the branch 😅 

```
k get helmrelease -A
NAMESPACE     NAME                    AGE   READY   STATUS
flux-system   kepler                  4d    True    Release reconciliation succeeded
flux-system   kube-prometheus-stack   90s   True    Release reconciliation succeeded
```

Could you revert this to main before merging?

Comment (nikimanoledaki) [reply clusters/flux-system/gotk-sync.yaml:11]: Sure! Will remember to revert the commit/change to point at `main` 👍

**Review by nikimanoledaki — APPROVED** [review]
Flux changes LGTM 👍

File: clusters/kube-prometheus-stack.yaml [hunk 3]
```diff
@@ -11,17 +11,19 @@ spec:
...
 kind: HelmRelease
 metadata:
-  name: prometheus
-  namespace: monitoring
+  name: kube-prometheus-stack
+  namespace: flux-system
 spec:
   interval: 24h
+  install:
+    createNamespace: true
```
[the remark this answers is not in the document]
Comment (nikimanoledaki) [reply clusters/kube-prometheus-stack.yaml:19]: 💯

File: clusters/flux-system/gotk-sync.yaml (hunk 2 above)
Comment (nikimanoledaki) [reply clusters/flux-system/gotk-sync.yaml:11]: Switching branches gave me an idea: we could use a branch strategy to test the different CNCF projects when we need to deploy them with Flux. 💡 

For example, we could move all the Falco-specific config (mainly the flux config for the synthetic workload and Falco itself) to its own `falco` branch. So, `main` contains all the basic components, and the branches contain specific components. 

Not something for now though, we can revisit this later, but wanted to share the idea 😄

Comment (rossf7) [reply clusters/flux-system/gotk-sync.yaml]: I like this idea and the PR based workflow for the projects is nice.

Eventually with a lot of projects branch management might get hard. Maybe at that point we could suspend the resources and then un-suspend when we want to deploy them? 🤔

**Pull request merged**